mlflow/store/model_registry/abstract_store.py

```diff
@@ -384,10 +385,15 @@ def copy_model_version(self, src_mv, dst_name):
             the cloned model version.
         """
         try:
-            self.create_registered_model(dst_name)
+            create_model_response = self.create_registered_model(dst_name)
+            eprint(f"Successfully registered model '{create_model_response.name}'.")
         except MlflowException as e:
             if e.error_code != ErrorCode.Name(RESOURCE_ALREADY_EXISTS):
                 raise
+            eprint(
```

Comment:
Curious, why do we use `eprint`?

Reply:
I took inspiration from this section: https://github.com/mlflow/mlflow/blob/4ae4e6fb610e2ab18d46e689faf080895b01f32f/mlflow/tracking/_model_registry/fluent.py#L91-L124